pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -5,6 +5,8 @@
 #include "rifcpp/rifImage.h"
 
 #include <RadeonProRender.h>
+#include <math/matrix.h>
+#include <math/mathutils.h>
```

Comment:
Is it necessary?

Reply:
hmm, weird. I think I had to add it to get rid of some compile errors. But now it also compiles without these two includes. I'll remove them.